subsys/bluetooth/host/iso_internal.h

```diff
@@ -11,6 +11,7 @@
 #include <bluetooth/iso.h>
 
 #define BT_ISO_DATA_PATH_DISABLED			0xFF
+#define BT_ISO_INVALID_BIG_HANDLE			0xFF
```

Comment:
Are these supposed to have the same value?

Reply:
They are just internal values, that represent some invalid value. It's okay that they are the same :)